ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/capella/SpecLogicCapella.java

```diff
@@ -86,7 +86,7 @@ public static SpecLogicCapella create(
       final SpecConfigCapella config, final SchemaDefinitionsCapella schemaDefinitions) {
     // Helpers
     final Predicates predicates = new Predicates(config);
-    final MiscHelpersBellatrix miscHelpers = new MiscHelpersBellatrix(config);
+    final MiscHelpersCapella miscHelpers = new MiscHelpersCapella(config);
```

Comment:
Seems like we normally use the previous fork class if it isn't changed. See how in `SpecLogicBellatrix` we still use Altair classes.

Reply:
My point is when Capella will add their helpers (there should be some, just was not done yet) we will have SpecLogicEIP4844 without Capella parent, so when processing block for example with EIP4844 speclogic, we could fail

Reply:
I could be wrong about Capella MiscHelpers should ever happen. And my example will be definitely float in our standard tests

Reply:
I see your point. But if there will be, we'll definitely notice if we forgot to extend from the new capella version since some reference tests will fail badly.

Reply:
For the record, there are no changes to MiscHelpers in Capella.  We've actually implemented all of the Capella spec changes now. :)